components/KiboShippingMethodForm.vue

```diff
@@ -0,0 +1,218 @@
+              {{ checkoutLineItemGetters.getProductName(product) }}
+            </div>
+
+            <div
+              v-for="option in checkoutLineItemGetters.getProductOptions(product)"
+              :key="option.attributeFQN"
+              class="content__props"
+            >
+              <span class="title"> {{ option.name }}: </span> {{ option.value }} <br />
+            </div>
+
+            <div class="content__props">
+              <span class="title"> Price: </span> ${{
+                checkoutLineItemGetters.getProductPrice(product)
+              }}
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="changeStore">
+      <span class="changeStore__location">Pickup in Store: </span><br />
+      <p class="changeStore__link" @click="handleStoreLocatorClick">
+        {{ cartItemPurchaseLocation ? "Change Store" : "Select Store" }}
+      </p>
+    </div>
+  </div>
+</template>
+
+<script>
+import { SfImage, SfSelect } from "@storefront-ui/vue"
+import { checkoutLineItemGetters } from "@/lib/getters"
+
+export default {
+  name: "KiboShipping",
+  components: {
+    SfImage,
+    SfSelect,
+  },
+  props: {
+    value: {
+      type: Object,
+      default: () => ({}),
+    },
+    cartItemPurchaseLocation: {
+      type: String,
+      default: "",
+    },
+    items: {
+      type: Array,
+      default: () => [],
+    },
+    shippingRates: {
+      type: Array,
+      default: () => [],
+    },
+  },
+  setup(props, context) {
+    const selectedShippingMethodCode = ref("")
+
+    const shippingMethod = ref({
+      shipItems: [],
+      pickupItems: [],
+    })
+
+    const updateField = (shippingMethodCode) => {
+      selectedShippingMethodCode.value = shippingMethodCode
+
+      const { shippingMethodName } = props.shippingRates.find(
+        (rate) => rate.shippingMethodCode === shippingMethodCode
+      )
+
+      context.emit("saveShippingMethod", { shippingMethodCode, shippingMethodName })
+    }
+
+    const handleStoreLocatorClick = () => {
+      context.emit("handleStoreLocatorClick")
+    }
+
+    return {
+      updateField,
+      checkoutLineItemGetters,
+      shippingMethod,
+      selectedShippingMethodCode,
+      handleStoreLocatorClick,
+    }
+  },
+}
+</script>
+
+<style lang="scss" scoped>
+@import "~@storefront-ui/shared/styles/components/templates/SfShipping.scss";
+
+.form {
+  border-top: 1px solid #eaeaea;
+
+  @include for-desktop {
+    width: calc(var(--spacer-base) * 17.54);
+  }
+
+  .sf-heading__title {
+    font-weight: bold;
+    padding: var(--spacer-sm) 0;
+  }
+
+  ::v-deep select:hover {
+    cursor: pointer;
+    background-color: black;
+    color: white;
+
+    option {
+      color: black;
+    }
+  }
+
+  .item {
+    display: flex;
+    flex-direction: column;
+
+    @include for-desktop {
+      width: calc(var(--spacer-base) * 17.54);
+    }
+
+    .productRow {
+      display: flex;
+      flex-direction: row;
```

Comment:
by default display flex has direction of row. No need to specify.

Reply:
Resolved.